dev-tools/formatters/intellij-formatter.xml

```diff
@@ -0,0 +1,77 @@
+<code_scheme name="OpenSearch" version="173">
```

Comment:
Do we want to add this in a .idea to pre-configure it?

Reply:
I don't know how to do that, but that sounds like a great follow up PR - want to take that on?